src/WebAPIContentService/Service/Interfaces/IMaterialService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using WebAPIContentService.Domain.Entities;
+
+namespace WebAPIContentService.Service.Interfaces
+{
+    public interface IMaterialService
+    {
+        Task<Material> GetMaterialByIdAsync(int id);
```

Comment:
```suggestion
        Task<Material?> GetMaterialByIdAsync(int id);
```

Mesmo motivo que falei acima, mas vale a pena confirmar não tenho certeza